webapp/crashstats/signature/tests/test_views.py

```diff
@@ -661,3 +663,16 @@ def test_signature_bugzilla(self, client, db, es_helper):
             < content.find("Related Crash Signatures")
             < content.find("Bugs for <code>OOM | small</code>")
         )
+
+    def test_get_fields(self, db, user_helper):
+        # Create anonymous user and get fields
+        user = AnonymousUser()
+        fields = get_fields(user)
+        len_public_fields = len(fields)
+        assert len(fields) > 0
+
+        # Create user with protected data access, get fields, and make sure there are
+        # more of them than if the user didn't have protected data access
+        user = user_helper.create_protected_user()
+        fields = get_fields(user)
+        assert len(fields) > len_public_fields
```

Comment:
Super search fields change over time, so I didn't want to create a test we had to update periodically. Instead, this asserts there are fields in the case where the user doesn't have protected data access and asserts that if the user does have protected data access, the user sees more fields than if the user didn't. That feels like a good enough set of assertions that aren't fragile.